terraso_backend/apps/project_management/migrations/0001_initial.py

```diff
@@ -46,13 +46,4 @@ class Migration(migrations.Migration):
                 violation_error_message="slug",
             ),
         ),
-        migrations.AddConstraint(
-            model_name="site",
```

Comment:
Question: Do we need a migration to remove this constraint? since it was already on main?

Reply:
Yeah, this should actually be dealt with in the next PR I think. Going to close this PR.